### ISSUE TITLE
Model predictions

### ISSUE THREAD

**Issue by mshearer0** [GoogleCloudPlatform/data-science-on-gcp#12]
I get the same results for different instances on pg328:

response={u'predictions': [{u'probabilities': [0.09236259013414383, 0.907637357711792], u'logits': [2.285122871398926], u'classes': 1, u'logistic': [0.907637357711792]}, {u'probabilities': [0.09236259013414383, 0.907637357711792], u'logits': [2.285122871398926], u'classes': 1, u'logistic': [0.907637357711792]}, {u'probabilities': [0.09236259013414383, 0.907637357711792], u'logits': [2.285122871398926], u'classes': 1, u'logistic': [0.907637357711792]}, {u'probabilities': [0.09236259013414383, 0.907637357711792], u'logits': [2.285122871398926], u'classes': 1, u'logistic': [0.907637357711792]}]}
probs=[0.907637357711792, 0.907637357711792, 0.907637357711792, 0.907637357711792]

The ontime probability=0.907637357712; the key reason is that this flight appears rather typical 
The ontime probability=0.907637357712; the key reason is that this flight appears rather typical 
The ontime probability=0.907637357712; the key reason is that this flight appears rather typical 
The ontime probability=0.907637357712; the key reason is that this flight appears rather typical 

Can you help with what i've done wrong?

Also, should factor be in range 1, len(probs) or range 0, len(probs) ?

for factor in xrange(0, len(probs)):
   impact = abs(probs[factor] - probs[0])
